internal/exec/runner.go

```diff
@@ -90,31 +90,58 @@ func newRunner(workDirPath string, input io.Reader, output io.Writer, options ..
 	return runner
 }
 
-func (r *runner) Version() error {
-	tabWriter := newTabWriter(r.output)
-	if _, err := fmt.Fprintf(tabWriter, "Version:\t%s\n", vars.Version); err != nil {
-		return err
-	}
-	if _, err := fmt.Fprintf(tabWriter, "Default protoc version:\t%s\n", vars.DefaultProtocVersion); err != nil {
-		return err
-	}
-	if _, err := fmt.Fprintf(tabWriter, "Go version:\t%s\n", runtime.Version()); err != nil {
-		return err
-	}
-	if vars.GitCommit != "" {
-		if _, err := fmt.Fprintf(tabWriter, "Git commit:\t%s\n", vars.GitCommit); err != nil {
+func (r *runner) Version(json bool) error {
+	out := &struct {
```

Comment:
No need to be a pointer, just do struct

Reply:
Done in 7668c8e6169c8b758e77784caedeff222c45fca1